src/lightning_lite/strategies/strategy.py

```diff
@@ -240,6 +240,7 @@ def save_checkpoint(
 
     def get_module_state_dict(self, module: Module) -> Dict[str, Union[Any, Tensor]]:
         """Returns model state."""
+        # TODO(lite): Integrate this into Lightning Lite
```

Comment:
What does this mean? It's in lightning lite already

Reply:
Yes, the api is here but it would be nice if we could call it in Lite, or expose it in some way. For example: #14816